Src/Couchbase.Linq/N1QlFunctions.Core.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Couchbase.Linq
+{
+    /// <summary>
+    /// Implements static helper methods for N1QL queries
+    /// </summary>
+    public static partial class N1QlFunctions
+    {
+        /// <summary>
+        /// Shortcut for creating an error for methods that are only supported in N1QL, not in .Net.
+        /// </summary>
+        private static Exception NotImplementedError()
+        {
+            return new NotImplementedException("This method may only be used for generating N1QL queries.");
```

Comment:
Ah, I think I got it. Yeah, I think not supported would make more sense. 

-Jeff